src/lib/markdown/operationLink.ts

```
@@ -0,0 +1,171 @@
+  return title || operation?.summary || operation?.operationId || ''
+}
+
+function createOperationLinkHtml(
+  href: string,
+  method: string,
+  title: string,
+): string {
+  return `<a href="${href}" class="OAOperationLink group/oaSidebarItem">`
+    + `<span class="OAOperationLink-badge OAMethodBadge--${method.toLowerCase()}">${method.toUpperCase()}</span>`
+    + `<span>${title}</span>`
+    + `</a>`
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)